pkg/server/updates.go

```diff
@@ -209,6 +210,12 @@ func (s *Server) checkForUpdates(ctx context.Context) {
 		return
 	}
 
+	// Ideally the updates server only returns the most relevant updates for us,
+	// but if it replied with an excessive number of updates, limit log spam by
+	// only printing the first few.
+	if len(r.Details) > updateMaxVersionsToReport {
+		r.Details = r.Details[:updateMaxVersionsToReport]
```

Comment:
We may want to get the last 3 rather than the first 3, since in my experience the most recent versions are listed last.